api/nakadi-event-bus-api.yaml

```diff
@@ -743,6 +743,56 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /subscriptions:
+    post:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        This endpoint creates a Subscription for EventType(s). The subscription is needed to be able to consume
+        events from EventType(s) in a high level way when Nakadi stores the offsets and manages the rebalancing of
+        consuming clients.
+        The subscription is identified by its parameters. If this endpoint is invoked several times with the same
+        subscription properties in body - the subscription will be created only once and for all calls it will
+        return the same subscription id.
+      parameters:
+        - name: subscription
+          in: body
+          description: Subscription to create
+          schema:
+            $ref: '#/definitions/Subscription'
+          required: true
+      responses:
+        '200':
```

Comment:
My only point here for not having the same response code is:
Let's imagine situation: one client creates a subscription with parameters A, he gets 201 and in response he gets a subscription with parameters A.
Now another client also tried to create subscription with the same owning_application, event_types and use_case but with parameters B. He will get 201 and in response he gets a subscription with parameters A. I think for client that will be quite unexpected, so we can at least differ this situation with different response code.

What do you think?